comixed-rest-api/src/main/java/org/comixed/net/ConvertComicsRequest.java

```diff
@@ -20,36 +20,44 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import java.util.List;
+import lombok.Getter;
+import lombok.Setter;
 import org.comixed.adaptors.ArchiveType;
 
+/**
+ * <code>ConvertComicsRequest</code> provides the HTTP request contents when converting comics.
+ *
+ * @author Darryl L. Pierce
+ */
 public class ConvertComicsRequest {
+  @Getter
+  @Setter
```

Comment:
If you want the getter and the setter for all the fields, this annotations can be put on the class.

Reply:
I would prefer to be specific with annotations like this, to avoid accidentally exposing a state variable that shouldn't be exposed. Annotating each field specifically helps make it explicit that it should be exposed with either a getter and/or a setter.

Reply:
It's good. I prefer to write more code to be specific and have my intentions clear, rather than writing less and leaving gaps that could lead to errors later :)